### DISCUSSION TITLE
2024-07-25 목요일

### DISCUSSION THREAD

**Discussion by Captain-Kim** [nbc-final-a10/TripBuddy#95] [기록보드]
2024-07-25 목요일 기록보드입니다.

## 여행가고십조(A10) 일일 공통 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 기술 면접 준비 | |
| 10:00 ~ 13:00 | 집중 개발 time | |
| 13:00 ~ 14:00 | 점심 식사 | |
| 14:00 ~ 15:00 | 일일 1차 정기 회의 | <img src="https://github.com/user-attachments/assets/47441dfe-b2d4-4042-addc-de46a0a8a9d1" width="300" height="300"/> |
| 15:00 ~ 18:00 | 개발 | |
| 18:00 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | 일일 2차 정기 회의(merge time) | |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | |

```
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 본인인증 외부 API 응답값 확인 및 테스트

// 사진
<img src="이미지 주소" width="300" height="300"/>

### 트러블 슈팅

- user id를 uuid로 받다 보니 segment가 지저분 해짐 -> login_id를 추가로 받는 것으로 해결

### 내일 계획

- supabase auth 비밀번호 찾기 기능 연결
```

**Comment by eunohhh** [comment]
## 오은 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 기술면접준비 | |
| 10:00 ~ 10:30 | 하루 일과 준비 | |
| 10:30 ~ 13:00 | 집중 개발 time| onboarding api 중심|
| 13:00 ~ 14:00 | 점심 식사 | |
| 14:00 ~ 15:00 | 일일 2차 정기 회의 | |
| 15:00 ~ 18:00 | 개발 | 버디성향/여정테마 훅으로 분리 |
| 18:00 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | 일일 3차 정기 회의 | |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | |

## 개인 기록보드 서식 (오은)

### 오늘 한 일

- 동적 모바일 헤더 작성(branch PR 완료)
- 온보딩관련 api 작성 진행중
- usePreferTheme 훅 작성 완료

### 트러블 슈팅

- 동적 모바일 헤더 작성시 nextjs 에서 제공하는 params 와 searchParams 를 스태틱 라우트에서는 쓸 수 없어, 직접 미들웨어를 조작하는 방식으로 해결
- 데이터설계에서 1차원배열보다는 객체로 변경하는 것이 좋다는 튜터님 피드백에 따라 구조 수정

### 내일 계획

- 스토리 페이지 시작

**Comment by hyunjeongkwak** [comment]
**오늘 한 일**

- 검색 페이지 컴포넌트 분리 중(title, calendar)
- 캘린더 start date, end date 데이터 저장 시도

**내일 할 일**

- 컴포넌트 분리 및 재사용 컴포넌트 적용(chip 등)
- 페이지 구조 정리 등 -> git push 하기
- (1280px UI 제작)

**Comment by Captain-Kim** [comment]
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 국가 선택 자료 구조 파악 및 타입 에러 수정 완료
- 국가 선택 컴포넌트 state명 등 시멘틱하게 코드 리팩토링
- 글쓰기 페이지 나머지 시퀀스 UI 초안 작성 완료

### 트러블 슈팅

- 2~3단계까지 내려가는 자료구조에서 타입 에러를 해결하기 어려움

### 내일 계획

- 글쓰기 페이지 기능 구현
- 마이페이지 UI 초안 제작

**Comment by slsl2** [comment]
## 개인 기록보드 (김민지)

### 오늘 한 일
- 채팅 목록 페이지 모바일 ui 작업
- 채팅 페이지 모바일 ui 작업

### 내일 계획
- 채팅 페이지 ui 작업 초안 완성
- 채팅 목록, 채팅 페이지 데스크탑 작업
- 채팅 기능 조금이라도 시작

**Comment by Kyara0324** [comment]
### 오늘 한 일
- Tutorial 페이지 퍼널패턴 적용
- Tutorial 페이지 폴더구조 정리
- 디자이너님 미팅 

### 내일 할 일
- reusable component를 어떻게 만들어야 좋을지 고민
- 튜토리얼 페이지 인디케이터 적용, 와이어프레임이 없으므로 대략적인 ui 구성
